src/app/components/SearchContainer/SearchContainer.jsx

```diff
@@ -29,6 +29,7 @@ class SearchContainer extends React.Component {
 
   componentDidMount() {
     this.loadSearch();
+    global.document.title = 'Search Results | ResearchNow | NYPL';
```

Comment:
This is getting set when I first load the site. Is there any way that the `SearchContainer` component is mounting before the page loads and then is not rendered?

Reply:
No, but I noticed there is a mistake going back to home. Fixing it now.